added missing documentation about case_insensitive param in wildcard … (#4581)
opensearch-project/documentation-website: _query-dsl/query-dsl/term.md

:--- | :---
`*` | Specifies all valid values.
`?` | Specifies a single valid value.

> `case_insensitive` | Parameter to indicate if the wildcard query should consider case. false to be case sensitive. true to be case insensitive. this is not required and will be case sensitive by default.

Reviewer: ```suggestion
`case_insensitive` | If `true`, the wildcard query is case insensitive. If `false`, the wildcard query is case sensitive. Optional. Default is `true` (case insensitive).
```

  }
}
```

> To search for terms with case sensitivity on:

Reviewer: ```suggestion
Use the following request for a case-sensitive term search:
```

```json
GET shakespeare/_search
{
  "query": {
    "wildcard": {
      "speaker": {
        "value": "H*Y",

> 		"case_insensitive": false

Reviewer: ```suggestion
	"case_insensitive": false
```
